src/Types/OwnedParameter.cs

```diff
@@ -0,0 +1,6 @@
+namespace Concordium.Sdk.Types;
+
+/// <summary>
+/// Parameter to the init function or entrypoint. Owned version.
```

Comment:
I think this "Owned" prefix does not make sense in C#.

It makes sense in Rust since there is a difference between Vec<u8> and &[u8], but in C# you in any case have this implicit indirection, so you can't really have &[u8] equivalent.

So I recommend removing all these Owned prefixes.